packages/apollo-server-fastify/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "apollo-server-fastify",
+  "version": "2.2.2",
+  "description": "Production-ready Node.js GraphQL server for Fastify",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apollographql/apollo-server/tree/master/packages/apollo-server-fastify"
+  },
+  "keywords": [
+    "GraphQL",
+    "Apollo",
+    "Server",
+    "Fastify",
+    "Javascript"
+  ],
+  "author": "opensource@apollographql.com",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/apollographql/apollo-server/issues"
+  },
+  "homepage": "https://github.com/apollographql/apollo-server#readme",
+  "engines": {
+    "node": ">=6"
+  },
+  "dependencies": {
+    "@apollographql/apollo-upload-server": "^5.0.3",
```

Comment:
Slight tweak to remove this direct dependency thanks to #2054.

Reply:
https://github.com/apollographql/apollo-server/pull/1971/commits/19d96320ddf295ff4b9d24d8359eb17490b9855d